sbs.py

```diff
@@ -28,6 +30,7 @@ def __init__(self, config):
         self.user = config.get('user', 'sysread')
         self.password = config.get('passwd', 'changethispassword')
         self.verify_ssl = config.get('verify_ssl', True)
+        self.timeout = config.get('timeout', None)
```

Comment:
Waarom geen default van 30 seconden als timeout niet geconfigureerd is?

Reply:
Hebben we via slack al even over gehad. Is kwestie van smaak. Mijn overweging was dat als je 'niets' expliciet opgeeft, dat je terugvalt op de standaard werkwijze. Die is voor requests timeout 'None' (Betekent: BLocking wait). Een default van 30 seconden kan maar zo te krap zijn, (bijvoorbeeld op dit moment op test met die enorme seed, die API duurt al langer dan 30 seconden. Dus dan is weer de vraag, wat moet dan de default zijn...
Mijn advies: Hanteer een timeout alleen als die expliciet is opgegeven.